Microsoft.Toolkit.Uwp.UI/Extensions/ListViewBaseExtensions.cs

```diff
@@ -0,0 +1,106 @@
+﻿using System.Windows.Input;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+
+namespace Microsoft.Toolkit.Uwp.UI
+{
+    /// <summary>
+    /// Provides attached dependency properties for the <see cref="ListViewBase"/> content element that allows
+    /// it to invoke a <see cref="ICommand"/> when clicked
+    /// </summary>
+    public static class ListViewBaseExtensions
+    {
+        /// <summary>
+        /// Attached <see cref="DependencyProperty"/> for binding an <see cref="ICommand"/> instance to a <see cref="ListViewBase"/>
+        /// </summary>
+        public static readonly DependencyProperty CommandProperty = DependencyProperty.RegisterAttached(
+            "Command",
```

Comment:
Added a noted that says it requires IsItemClickEnabled needs to be sets for this extension command to work